app/src/main/java/com/dinesh/ficasso/Ficasso.java

```diff
@@ -11,23 +11,16 @@
     public static Ficasso ourInstance = new Ficasso();
```

Comment:
Remove your says Created by

Reply:
It is good instantiate only when needed, So you can move instantiation to load method

Reply:
i didn't get , can you elobrate it